lib/src/test/java/nl/knaw/dans/dansschema/DdmVersionChangesTest.java

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (C) 2022 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.dansschema;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.xml.sax.SAXParseException;
+
+import static nl.knaw.dans.dansschema.XmlReader.readXmlString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class DdmVersionChangesTest {
+    private static final String xsdDir = "src/main/resources/md/ddm/";
+    private static final String ddmNamespaceV1 = "http://easy.dans.knaw.nl/schemas/md/ddm/";
+    private static final String ddmNamespaceV2 = "http://schemas.dans.knaw.nl/dataset/ddm-v2/";
+    private static SchemaValidator ddmValidatorV2;
+    private static SchemaValidator ddmValidatorV1;
+
+    @BeforeAll
+    public static void initValidators() throws Exception{
+        ddmValidatorV2 = new SchemaValidator(xsdDir + "v2/ddm.xsd");
+        ddmValidatorV1 = new SchemaValidator(xsdDir + "v1/ddm.xsd");
+    }
+
+
+    @Test
+    public void xml_without_root_namespace_should_not_parse() {
+        assertThatThrownBy(() -> (readXmlString("<ddm:DDM/>")).normalize())
+            .isInstanceOf(SAXParseException.class)
+            .hasMessage("The prefix \"ddm\" for element \"ddm:DDM\" is not bound.");
+    }
+
+    @Test
+    public void v1_should_require_schemas_dans_knaw_nl_name_space_() throws Exception {
+        var result = ddmValidatorV1.validateString("<ddm:DDM  xmlns:ddm='http://schemas.dans.knaw.nl/dataset/ddm-v2/'/>");
+        assertThat(result).hasSize(1);
+        assertThat(result.get(0))
+            .hasMessage("cvc-elt.1.a: Cannot find the declaration of element 'ddm:DDM'.");
+    }
+
+    @Test
+    public void v2_should_require_easy_dans_knaw_nl_name_space() throws Exception {
+        var result = ddmValidatorV2.validateString("<ddm:DDM xmlns:ddm='http://easy.dans.knaw.nl/schemas/md/ddm/'/>");
+        assertThat(result).hasSize(1);
+        assertThat(result.get(0))
+            .hasMessage("cvc-elt.1.a: Cannot find the declaration of element 'ddm:DDM'.");
+    }
+
+    @Test
+    public void v1_should_allow_additional_xml_in_root() throws Exception {
```

Comment:
The test name does not explain well what is tested. What _actually_ seems to be tested is that an empty ddm is not valid. A good additional test would be to have a DDM with the minimally required content and show that is passes.